src/main/scala/uk/gov/homeoffice/drt/prediction/FeaturesWithOneToManyValues.scala

```diff
@@ -0,0 +1,21 @@
+package uk.gov.homeoffice.drt.prediction
+
+import uk.gov.homeoffice.drt.prediction.Feature.{Feature, OneToMany, Single}
+
+object Feature {
```

Comment:
as there is sealed trait for Feature do we need Single and OneToMany in object Feature ?